frontend/src/context/Metrics/metricsSlice.tsx

```diff
@@ -89,6 +95,51 @@ const findKeyByValues = (arrayA: { [key: string]: string }[], arrayB: string[]):
   }
   return `The value of ${matchingKeys} in imported json is not in dropdown list now. Please select a value for it!`
 }
+
+const setSelectUsers = (users: string[], importedCrews: string[]) =>
+  users?.filter((item: string) => importedCrews?.includes(item))
```

Comment:
do u really need '?' ,because u defined type of users and importedCrews.

Reply:
‘?’ after users and targetfileds could be deleted because we have handled this, but '?' after import data cannot be deleted, because importedCrews could be undefined if we delete it in json file